BundleClient/app/src/main/java/com/ddd/utils/BundleUtils.java

```diff
@@ -155,13 +155,14 @@ public static boolean doContentsMatch(UncompressedPayload.Builder a, Uncompresse
     public static void writeUncompressedPayload(UncompressedPayload uncompressedPayload, File targetDirectory,
                                                 String bundleFileName) {
         String bundleId = uncompressedPayload.getBundleId();
-        String bundleFilePath = targetDirectory.getAbsolutePath() + "/" + bundleId;
+        String bundleFilePath = targetDirectory.getAbsolutePath() + File.separator + bundleId;
```

Comment:
this should be var bundleFilePath = targetDirectory.getAbsolutePath().resolve(bundleId);